data/core/units/elves/Ranger.cfg

```diff
@@ -16,7 +16,7 @@
     advances_to=Elvish Avenger
     cost=38
     usage=mixed fighter
-    description= _ "Though a man might spend years in the forest, he will never shake the feeling that he is a guest in a realm of which he is not truly a part. With elves, this is quite reversed. Any elf that studies the lore of the woods rapidly becomes a master of them. This, combined with a considerable skill at archery and swordsmanship, is very useful in warfare."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}
+    description= _ "The silent watchers of the forests, rangers are masters of stealth and reconnaissance. Many are explorers and loners, preferring to travel alone along the paths between the towering trees and among the unspoiled natural beauty of their homes of which they are fiercely protective. While they possess considerable skill in both blade and bow, rangers are a little less adept at fighting other warriors on even terms, preferring instead to ambush their enemies from the shadows in the woods. A surprise attack from rangers can cripple even a force many times their size, and attacking armies often choose to withdraw rather than risk a game of guerrilla warfare with rangers in the forests."+{SPECIAL_NOTES}+{SPECIAL_NOTES_AMBUSH}
```

Comment:
Check punctuation or reword long sentences?

You're warning that Rangers are not a good choice for front-line troops, but take so long to get to the point it's hard to see they make up for it by being stealthy. Maybe reverse the order and present their stealthiness, first, then warn they're not so good in a stand-up fight? I do like the warning at the end.

Reply:
I only use one clause to suggest that Rangers are not as strong as your typical warriors, so I don't think that's so long at all.

Reply:
What I'm getting at is that it might be more clear to lead off with the positive effect the player should be considering (stealth) and follow up with the warning that they're a bit weaker combatant as a result of their concentration on being so stealthy. It's not clear that a Ranger can be devistating, when using stealth, but will have a rough time if caught in the open.

Reply:
On the flip side, I think that saying that they are weaker in personal combat, but can use stealth instead also is a reasonable way to state the same thing, and flows better into the following sentences.